### ISSUE TITLE
BBS demo

### ISSUE THREAD

**Issue by billsimon** [ExileInParadise/cortex84#1]
Just wondered if you have a public BBS online that demonstrates this fork?

**Comment by ExileInParadise** [comment]
Yes I do - ssh to port 6022 of animaexmachina.sytes.net as user new to signup.
Example from a Linux prompt: ssh -p 6022 new@animaexmachina.sytes.net
To login after signup, ssh with the command above, replacing new with the login name you chose.
The BBS code primarily supports 80x25 Unicode and Code Page 437 characters and ANSI escape codes. ANSI-compatible terminals on many retro computers that support 80 columns usually work.
My handle on the board is aXoN - please feel free to send me a message or leave feedback there as well. There is a cortex84 message area on the board as well for just talking about this fork and running instance. Hope this helps and thanks for the question. I will close out this issue once I write up a README.md for the fork and put it in the top of the repo. Probably the main differences to look for are the Elite-x/84 native game on the games menu (x84/default/elitex84.py and intro.txt), an update to the message area code to allow menus to call into specific message areas, some changes to menu code itself for various areas, a set of unified menu and section ansis using a common font and changing colors for items. Most of the changes in the fork were cleaning up pylint warnings.

**Comment by davestephens** [comment]
I don't suppose this is still online?

**Comment by ExileInParadise** [comment]
Yep, it still is. If you have any trouble with the instructions above, please let me know.

**Comment by davestephens** [comment]
Hm, doesn't seem to be accessible from London, just hangs when trying to SSH in.
